sdk-ids/src/lib.rs

```diff
@@ -1,121 +1,121 @@
+    solana_address::declare_id!("BPFLoaderUpgradeab1e11111111111111111111111");
 }
 
 pub mod compute_budget {
-    solana_pubkey::declare_id!("ComputeBudget111111111111111111111111111111");
+    solana_address::declare_id!("ComputeBudget111111111111111111111111111111");
 }
 
 pub mod config {
-    solana_pubkey::declare_id!("Config1111111111111111111111111111111111111");
+    solana_address::declare_id!("Config1111111111111111111111111111111111111");
 }
 
 pub mod ed25519_program {
-    solana_pubkey::declare_id!("Ed25519SigVerify111111111111111111111111111");
+    solana_address::declare_id!("Ed25519SigVerify111111111111111111111111111");
 }
 
 pub mod feature {
-    solana_pubkey::declare_id!("Feature111111111111111111111111111111111111");
+    solana_address::declare_id!("Feature111111111111111111111111111111111111");
 }
 
 /// A designated address for burning lamports.
 ///
 /// Lamports credited to this address will be removed from the total supply
 /// (burned) at the end of the current block.
 pub mod incinerator {
-    solana_pubkey::declare_id!("1nc1nerator11111111111111111111111111111111");
+    solana_address::declare_id!("1nc1nerator11111111111111111111111111111111");
 }
 
 pub mod loader_v4 {
-    solana_pubkey::declare_id!("LoaderV411111111111111111111111111111111111");
+    solana_address::declare_id!("LoaderV411111111111111111111111111111111111");
 }
 
 pub mod native_loader {
-    solana_pubkey::declare_id!("NativeLoader1111111111111111111111111111111");
+    solana_address::declare_id!("NativeLoader1111111111111111111111111111111");
 }
 
 pub mod secp256k1_program {
-    solana_pubkey::declare_id!("KeccakSecp256k11111111111111111111111111111");
+    solana_address::declare_id!("KeccakSecp256k11111111111111111111111111111");
 }
 
 pub mod secp256r1_program {
-    solana_pubkey::declare_id!("Secp256r1SigVerify1111111111111111111111111");
+    solana_address::declare_id!("Secp256r1SigVerify1111111111111111111111111");
 }
 
 pub mod stake {
     pub mod config {
-        solana_pubkey::declare_deprecated_id!("StakeConfig11111111111111111111111111111111");
+        solana_address::declare_deprecated_id!("StakeConfig11111111111111111111111111111111");
     }
-    solana_pubkey::declare_id!("Stake11111111111111111111111111111111111111");
+    solana_address::declare_id!("Stake11111111111111111111111111111111111111");
 }
 
 pub mod system_program {
-    solana_pubkey::declare_id!("11111111111111111111111111111111");
+    solana_address::declare_id!("11111111111111111111111111111111");
 }
 
 pub mod vote {
-    solana_pubkey::declare_id!("Vote111111111111111111111111111111111111111");
+    solana_address::declare_id!("Vote111111111111111111111111111111111111111");
 }
 
 pub mod sysvar {
     // Owner pubkey for sysvar accounts
-    solana_pubkey::declare_id!("Sysvar1111111111111111111111111111111111111");
+    solana_address::declare_id!("Sysvar1111111111111111111111111111111111111");
```

Comment:
The comment above says "Owner pubkey" – wondering if it should be "Owner address".

Reply:
Ah nice find, thanks!